extra/io/files/trash/windows/windows.factor

```diff
@@ -51,6 +52,7 @@ PRIVATE>
 
 M: windows send-to-trash ( path -- )
     [
+        absolute-path [ dup CHAR: / = [ drop CHAR: \\ ] when ] map
```

Comment:
Perhaps we should call ``normalize-path`` here instead?

Reply:
I avoided `normalize-path`, because it returns a UNC path, and the documentation says that "SHFileOperation fails on any path prefixed with "\?"."

Actually, please skip this commit: it crashes for me at the moment with a memory protection fault - the original version, the one I submitted in this PR and the one with `normalize-path`, they all crash.

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/shellapi/ns-shellapi-shfileopstructa